packages/ui/components/centered-container.tsx

```diff
@@ -0,0 +1,7 @@
+import { type PropsWithChildren } from "react";
+
+export const CenteredContainer = (props: PropsWithChildren) => (
```

Comment:
we don't need this component anymore, this can be simply done by using `container` class from `tailwind` 😅